backend/src/test/java/ai/verta/modeldb/DatasetTest.java

```diff
@@ -1414,6 +1413,9 @@ public void getLastExperimentByDataset() throws InterruptedException {
           experimentRun.getDateUpdated(),
           response.getExperimentRun().getDateUpdated());
 
+      ReconcilerInitializer.initialize(
+          testConfig, services, testConfig.getTestJdbi(), handleExecutor);
+
```

Comment:
`LastExperimentByDatasetId` endpoint called based on update timestamp. Before changes we have inline update parent timestamp code  but now we have reconcile utils so need to run it before calling `LastExperimentByDatasetId`